docs/examples/evaluation/multi_modal/multi_modal_rag_evaluation.ipynb

```diff
@@ -0,0 +1,1242 @@
+{
```

Comment:
<p>Having just come from the multimodal RAG eval blog post, I thought we would be doing LMM-as-a-judge here. Let's just add a note to mention that as a good next step here as well? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/8852/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>